grizli/utils.py

```diff
@@ -5751,6 +5765,9 @@ def fetch_config_files(ACS=False, get_sky=True, get_stars=True, get_epsf=True):
 
     tarfiles += [f'{BASEURL}/ACS.WFC.CHIP1.Stars.conf', 
                  f'{BASEURL}/ACS.WFC.CHIP2.Stars.conf']
+
+    if get_jwst:
+        tarfiles += [f'{BASEURL}/jwst-grism-conf.tar.gz']
```

Comment:
Good!  Could you do one more thing and make fetching wfc3 not implicit?  That is, add a `fetch_wfc3` option with default `True` and append the `V4.32` files to `tarfiles` if it's set.  And then also the `ACS.WFC.CHIP?.Stars.conf` rows should be within the `ACS` block.  And finally I guess the option `ACS` should be `get_acs` and you can provide some back compatibility with 

```python
def fetch_config_files(..., **kwargs):
    ...
    if 'ACS' in kwargs:
        fetch_acs = kwargs['ACS']
    ...
```

Reply:
NB, the [installation docs](https://github.com/gbrammer/grizli/blob/master/docs/grizli/install.rst) would need to be updated with the new `get_acs` and also `get_jwst` keywords.

Reply:
Fixed. Is the test working now? 81e00bf075d9ff4250ebd77ab5df0ae2ba41629b